google/cloud/storage/blob.py

```diff
@@ -1106,6 +1110,77 @@ def download_as_string(
         )
         return string_buffer.getvalue()
 
+    def download_as_string(
```

Comment:
`download_as_text` would be a clearer name for this method, given the historic ambiguity of the term "string" in Python.